### PR TITLE
[5.0] osprofiler support

### PR DESCRIPTION
**Pull request by toabctl**

### CODE REVIEW

File: chef/data_bags/crowbar/template-keystone.json [hunk 1]
```diff
@@ -171,14 +171,20 @@
...
+        "connection_string": "messaging://"
       }
     }
   },
   "deployment": {
     "keystone": {
       "crowbar-revision": 0,
       "crowbar-applied": false,
-      "schema-revision": 205,
+      "schema-revision": 207,
```
Comment (cmurphy) [inline chef/data_bags/crowbar/template-keystone.json]: Wrong number

Comment (toabctl) [reply chef/data_bags/crowbar/template-keystone.json]: @cmurphy no. there are 2 migrations involved.

Comment (toabctl) [reply chef/data_bags/crowbar/template-keystone.json]: @cmurphy friendly ping

Comment (cmurphy) [reply chef/data_bags/crowbar/template-keystone.json]: Sorry! Thanks for the reminder

File: chef/data_bags/crowbar/migrate/keystone/206_add_osprofiler.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+def upgrade(template_attrs, template_deployment, attrs, deployment)
+  unless attrs.key? "osprofiler"
+    attrs["osprofiler"] = template_attrs["osprofiler"]
+    unless defined?(@@osprofiler_hmac_keys)
```
Comment (dirkmueller) [inline chef/data_bags/crowbar/migrate/keystone/206_add_osprofiler.rb]: shouldn't this be @@keystone_osprofiler_hmac_keys ?

Comment (toabctl) [reply chef/data_bags/crowbar/migrate/keystone/206_add_osprofiler.rb]: Fixed

**Pull request merged**